### ISSUE TITLE
Support starting database / code host from command-line

### ISSUE THREAD

**Issue by per-samuelsson** [Starcounter/Starcounter#2950]
I've discussed this some with @alemoi, and we have also discussed it previously (I know @k-rus and me discussed it some year ago).

The proposal is we support it via `staradmin`, since thats the tool that already support stopping of databases and/or hosts.

**Comment by alemoi** [comment]
:+1: Basically, I would like to have the exact behavior of `star.exe -d=default appname.exe` but without `appname.exe` (e.g. so it will create the database)

**Comment by per-samuelsson** [comment]
During some discussion today, me, @alemoi and @dan31 decided that we'll want to use this in the installer, to initialize the environment (using an unelevated process session) and that we want to back that by `staradmin start db` (including implicit creation of default database if it's not created) and also support `staradmin start server`. The former should include the functionality of the later.

My aim is to do this within the nearest week or so.

**Comment by per-samuelsson** [comment]
At the end of this:

1. Update documentation on `starcounter.io`
2. Change TestApps to use this rather than the "empty app" workaround.

**Comment by per-samuelsson** [comment]
> At the end of this: [...]

And update the changelog obv! :smile:

**Comment by per-samuelsson** [comment]
The functionality of this is done, merged into main trunk with https://github.com/Starcounter/Starcounter/commit/1c928c6d7c7bba03d81946ab5cd862042d5fa08d.

With this, we now support `staradmin start server` and `staradmin start db`. The later is now the base of `star app.exe`, i.e. it will automatically create the database if it doesn't exist and before starting it.

> Change TestApps to use this rather than the "empty app" workaround.

Done.

> And update the changelog obv!

Done.

> Update documentation on starcounter.io

Doing that, and then closing this issue.

**Comment by alemoi** [comment]
Super good! Then i will catch up with this once have a time slot!

**Comment by per-samuelsson** [comment]
>> Update documentation on starcounter.io

> Doing that, and then closing this issue.

Done too: [staradmin CLI | Start command](http://starcounter.io/guides/tools/staradmin/#start-command).
